src/frontend/App.js

```diff
@@ -26,6 +26,8 @@ const log = debug("mapeo:App");
 const NAV_STORE_KEY = "@MapeoNavigation@9";
 const ERROR_STORE_KEY = "@MapeoError";
 
+const DEEPLINK_URI_PREFIX = "mapeo://";
```

Comment:
Is there a good place to keep constants like this? Thinking it'd be useful for anything that may generate a deeplink to have access to this (such as the Send Join Request screen)

Reply:
Yes, great idea!

Reply:
created `/src/frontend/constants.ts` for this